### PR TITLE
feat(deequ): added integration with Deequ

### PR DESCRIPTION
**Pull request by RonBarabash**

### CODE REVIEW

**Comment by giladw** [comment]
looks cool! a great feature :)

File: src/main/scala/com/yotpo/metorikku/metric/stepActions/DataQualityCheck.scala [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  def run(dataFrame: DataFrame): Unit = {
+    val checkList: Seq[Check] = checks.map {
+      case "uniqueness" => new Check(CheckLevel.Error, "uniqueness checks").isUnique(column)
+      case "completeness" => new Check(CheckLevel.Error, "completeness checks").isComplete(column)
+      case _ => new Check(CheckLevel.Error, "completeness checks").isComplete(column)
+    }
+    val verificationRunBuilder = VerificationSuite().onData(dataFrame).addChecks(checkList)
+    val verificationResult = verificationRunBuilder.run()
+    if (verificationResult.status == CheckStatus.Success) {
+      print("The data passed the test, everything is fine!")
```
Comment (lyogev) [inline src/main/scala/com/yotpo/metorikku/metric/stepActions/DataQualityCheck.scala:21]: Please use the logger

File: src/main/scala/com/yotpo/metorikku/metric/stepActions/DataQualityCheck.scala [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+      val resultsForAllConstraints = verificationResult.checkResults
+        .flatMap { case (_, checkResult) => checkResult.constraintResults }
+
+      resultsForAllConstraints
+        .filter {
+          _.status != ConstraintStatus.Success
+        }
+        .foreach { result =>
+          print(s"${result.constraint} failed: ${result.message.get}")
```
Comment (lyogev) [inline src/main/scala/com/yotpo/metorikku/metric/stepActions/DataQualityCheck.scala:33]: You're just printing... what are your plans with this?

**Comment by tooptoop4** [comment]
fixes https://github.com/YotpoLtd/metorikku/issues/191 :)

**Pull request closed without merging**